### ISSUE TITLE
Snyk reports a new version when at latest

### ISSUE THREAD

**Issue by dominykas** [snyk/snyk#115]
- `node -v`: `v6.11.5`
- `npm -v`: `5.5.1`

### Actual behaviour

```
✔ ~/devel/projects/wave2/ui [change-loss|✔] 
17:11 $ snyk config set disable-analytics=1
disable-analytics updated



┌────────────────────────────────────────────┐
│ Update available: 1.52.1 (current: 1.49.5) │
│ Run npm install -g snyk to update.         │
└────────────────────────────────────────────┘

✔ ~/devel/projects/wave2/ui [change-loss|✔] 
17:12 $ snyk --version
1.54.1
```

**Comment by maban** [comment]
Hi! This is a new one! This is showing some very odd information – it's reporting your current version as `1.49.5`, and then as `1.54.1`, and the latest version as `1.52.1` despite it being at least `1.54.1`.

What happens when you run `npm install -g snyk`? Do you still get that message?

**Comment by dominykas** [comment]
This was just after installing it for the first time, oddly. I did reinstall and I don't seem to be able to repro anymore...

**Comment by maban** [comment]
Very weird, please shout if it happens again!

**Comment by dominykas** [comment]
Just got this again:

```
21:52 $ npm i -g snyk
/Users/dominykas/.nvm/versions/node/v8.9.4/bin/snyk -> /Users/dominykas/.nvm/versions/node/v8.9.4/lib/node_modules/snyk/cli/index.js
+ snyk@1.69.10
added 1 package and updated 5 packages in 27.541s
✔ ~/devel/projects/jjj [master|✔] 
21:53 $ vim .snyk 
✔ ~/devel/projects/jjj [master ↓·10|✚ 1] 
21:53 $ snyk test --dev

Testing /Users/dominykas/devel/projects/jjj...
✗ High severity vulnerability found on shelljs@0.8.1
- desc: Command Injection
- info: https://snyk.io/vuln/npm:shelljs:20140723
- from: jjj@2.3.0 > shelljs@0.8.1
Fix: None available. Consider removing this dependency.

Organisation: dominykas
Package manager: npm
Target file: package.json
Open source: no
Local Snyk policy found

Tested 207 dependencies for known vulnerabilities, found 1 vulnerability, 1 vulnerable path.

Run `snyk wizard` to address these issues.

You are approaching your limit of private tests this billing period.


┌────────────────────────────────────────────┐
│ Update available: 1.69.9 (current: 1.69.7) │
│ Run npm install -g snyk to update.         │
└────────────────────────────────────────────┘

✘-1 ~/devel/projects/jjj [master ↓·10|✚ 1] 
21:53 $ snyk --version
1.69.10
```

This happened on the _first_ run after the install (and does not happen on the second run).

**Comment by dominykas** [comment]
I've been using `update-notifier` in other projects, and I suspect it's just because of how it checks for the versions asynchronously.
